lib/sync-DataSetModel.js

```diff
@@ -400,7 +400,7 @@ var self = {
       syncUtil.doLog(dataset.id, 'verbose', 'doSyncList cb ' + ( cb !== undefined) + ' - Global Hash (prev :: cur) = ' + previousHash + ' ::  ' + globalHash);
 
       datasetClient.dataHash = globalHash;
-      self.datasetClientRecords[datasetClient.id] = recOut;
+      // self.datasetClientRecords[datasetClient.id] = recOut;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)